Bazel build is broken

```
% bazel build cmd:jsonnet
INFO: Found 1 target...
ERROR: /Users/shimin/jsonnet/core/BUILD:32:1: C++ compilation of rule '//core:parser' failed: osx_cc_wrapper.sh failed: error executing command external/bazel_tools/tools/cpp/osx_cc_wrapper.sh '-D_FORTIFY_SOURCE=1' -fstack-protector -fcolor-diagnostics -Wall -Wthread-safety -Wself-assign -fno-omit-frame-pointer '-std=c++0x' -iquote . -iquote ... (remaining 29 argument(s) skipped): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
In file included from core/parser.cpp:29:
In file included from core/desugarer.h:24:
core/vm.h:21:10: fatal error: 'libjsonnet.h' file not found
#include "libjsonnet.h"
         ^
1 error generated.
Target //cmd:jsonnet failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 1.412s, Critical Path: 1.24s
```
core/vm.h is in the //core package whereas libjsonnet.h is in the //include package.

I get a different problem in fact:

```
dcunnin@tyrion:~/jsonnet$ bazel build cmd:jsonnet
INFO: Found 1 target...
ERROR: /home/dcunnin/jsonnet/core/BUILD:32:1: C++ compilation of rule '//core:parser' failed: namespace-sandbox failed: error executing command /home/dcunnin/.cache/bazel/_bazel_dcunnin/27e56e06b2deafe67d85288b6bda6714/jsonnet/_bin/namespace-sandbox ... (remaining 44 argument(s) skipped).
In file included from core/desugarer.cpp:20:0:
core/desugarer.h:24:16: fatal error: vm.h: No such file or directory
 #include "vm.h"
                ^
compilation terminated.
Target //cmd:jsonnet failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 0.746s, Critical Path: 0.64s
```

```
$ bazel version
Build label: 0.2.1rc1-jdk7
Build target: bazel-out/local_linux-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Mon Mar 21 20:24:52 2016 (1458591892)
Build timestamp: 1458591892
Build timestamp as int: 1458591892
```

--- a/core/BUILD
+++ b/core/BUILD
@@ -46,6 +46,7 @@ cc_library(
         ":lexer",
         "//stdlib:std",
     ],
+    includes = [".", "../include"],
 )
 
 cc_test(
@@ -80,7 +81,7 @@ cc_library(
         "//include:libjsonnet",
     ],
     linkopts = ["-lm"],
-    includes = ["."],
+    includes = [".", "../include"],
 )
 
 cc_library(
@@ -90,6 +91,7 @@ cc_library(
         ":jsonnet-common",
         "//include:libjsonnet",
     ],
+    includes = [".", "../include"],
 )
 
 cc_test(

--- a/cpp/testdata/BUILD
+++ b/cpp/testdata/BUILD
@@ -9,8 +9,8 @@ jsonnet_to_json_test(
     name = "example_test",
     src = "example.jsonnet",
     golden = "example_golden.json",
-    jsonnet = "//:jsonnet",
-    std = "//:std",
+    jsonnet = "//cmd:jsonnet",
+    std = "//stdlib",
 )
 
 jsonnet_to_json_test(
@@ -18,8 +18,8 @@ jsonnet_to_json_test(
     src = "importing.jsonnet",
     golden = "importing_golden.json",
     imports = ["."],
-    jsonnet = "//:jsonnet",
-    std = "//:std",
+    jsonnet = "//cmd:jsonnet",
+    std = "//stdlib",
 )
 
 jsonnet_to_json_test(
@@ -27,8 +27,8 @@ jsonnet_to_json_test(
     src = "invalid.jsonnet",
     golden = "invalid.out",
     error = 1,
-    jsonnet = "//:jsonnet",
-    std = "//:std",
+    jsonnet = "//cmd:jsonnet",
+    std = "//stdlib",
 )
 
 filegroup(

--- a/stdlib/BUILD
+++ b/stdlib/BUILD
@@ -1,8 +1,8 @@
 package(default_visibility = ["//visibility:public"])
 
 filegroup(
-  name = "stdlib",
-  srcs = ["std.jsonnet"],
+    name = "stdlib",
+    srcs = ["std.jsonnet"],
 )
 
 cc_library(

--- a/stdlib/BUILD
+++ b/stdlib/BUILD
@@ -1,5 +1,10 @@
 package(default_visibility = ["//visibility:public"])
 
+filegroup(
+  name = "stdlib",
+  srcs = ["std.jsonnet"],
+)
+
 cc_library(
     name = "std",
     hdrs = ["std.jsonnet.h"],

